src/platforms/PolygonId/usePolygonIdCardProps.tsx

```diff
@@ -0,0 +1,22 @@
+import { Circle, Img, useColorModeValue } from "@chakra-ui/react"
+import platforms from "platforms/platforms"
+import { GuildPlatform, PlatformName } from "types"
+
+const usePolygonIdCardProps = (guildPlatform: GuildPlatform) => {
+  const bgColor = useColorModeValue("gray.700", "gray.600")
+
+  const platformGuildData = guildPlatform.platformGuildData
+
+  return {
+    name: "PolygonID proofs",
+    type: "POLYGON_ID" as PlatformName,
+    image: platformGuildData?.imageUrl ?? (
```

Comment:
Users won't be able to modify the platform's image, so you can just use `platforms.POLYGON_ID.imageUrl` here